src/integrationTest/resources/zipcontents/mismatching_pdfs/metadata.json

```diff
@@ -22,11 +22,11 @@
       },
       {
         "document_control_number": "1111001",
-        "scanning_date": "2018-06-23T00:00:00.000Z",
+        "scanning_date": "2018-06-23T02:10:30.123Z",
         "ocr_accuracy": "test",
         "manual_intervention": "test",
         "next_action": "return",
-        "next_action_date": "2018-06-23T00:00:00.000Z",
+        "next_action_date": "2018-06-24T00:00:00.000Z",
```

Comment:
Why the date change?

Reply:
it doesn't need to change, It can be same value. 
I just want to show one date field value where hour, minutes, seconds and milliseconds are zeros.

Reply:
it was zeros. day changed from 23 to 24